src/libData/AccountData/Account.cpp

```diff
@@ -44,6 +43,52 @@ void Account::InitStorage()
     }
 }
 
+void Account::InitContract(const vector<unsigned char>& data)
+{
+    if (data.empty())
+    {
+        m_initValJson = Json::arrayValue;
+        return;
+    }
+    Json::CharReaderBuilder builder;
+    std::unique_ptr<Json::CharReader> reader(builder.newCharReader());
+    Json::Value root;
+    string dataStr(data.begin(), data.end());
```

Comment:
Is it necessary to make a copy here? I didn't try but maybe directly passing `data` to `reader->parse` also works

Reply:
Just tried directly passing it and it doesn't work cause the type issue.

The following way works when using `reinterpret_cast`
```
reader->parse(reinterpret_cast<const char *>(data.data()), reinterpret_cast<const char *>(data.data()) + data.size(), &root, &errors)
```
 Any opinion on this?